content/blog/lifehacking/2017-12-01_freedom_and_productivity.md

```diff
@@ -0,0 +1,199 @@
+gives head space for other matters; on the other side, it means that there are
+no conflicts in thinking, less freedom of choice, no attempts to solutions that
+are more suited to different groups and contexts, and which ultimately could be
+better overall.
+
+## The fallacies of mimicry
+
+Let's explore another common objection that I call the fallacies of mimicry: as
+if by imitating the actions and habits of so-called successful people, you
+could get their success too. Warren Buffet reads a lot of hours a day; he is
+insanely rich; so if I read a lot of hours a day, then I'll become rich,
+somehow? I think this kind of reasoning is completely flawed, as it inverts the
+consequences with the premises. Notably, when you get successful enough (which
+seems to translate here by "you don't have to worry about money anymore"), you
+get this great amount of free time that gives you the possibility to read.
+There's another quirk in that thinking: different methods, techniques, habits
+apply to different people in different contexts, and there's no one unique
+solution that works for all. So repeating the actions of others might not be
+suited for your particular situation, and it could show useless.
+
+That being said, I think there's something else that makes these
+recommendations very appealing; in the end, it's not about reproducing these
+successful stories but much more about having a high-end lifestyle, in which
+you *take* time for some particular actions and habits, instead of feeling that
+you *lack* time for them. It is much more about being conscious of what really
+matters to you, what you really want to do, rather than getting rich and
+famous. This is not so much about the search of success than the search of
+happiness.
+
+This also raises another point I wonder about: is the seek of productivity only
+a matter of the priviledged? If you consider something like [Maslow's hierarchy
+of needs](https://en.wikipedia.org/wiki/Maslow's_hierarchy_of_needs),
+productivity would probably belong to the top level (self-actualization), since
+it assumes all other issues in your life have been addressed first. If we put
+aside the working life aspect, a great amount of knowledge can also apply to
+one's personal life and free time. Yet, it feels it's a sign that "success"
+compounds, to some extent: the less you have to worry about universal basic
+needs, the more you can have time to read, learn, get better and enhance
+yourself, which in turn makes you more suited to reach whatever goal you aim
+for. It doesn't only happen with productivity, but also with education: kids
+from richer families tend to perform better than kids from poorer families, etc
+(there are studies I don't have links to that try to prove this affirmation).
+
+## To seek the whys
+
+There's always something else that's tickling me about those posts: they seem
+to confound the means and the ends. Being productive isn't an end per se, it is
```

Comment:
s/confound/confuse? I'm not used to reading the word "confound" but it might just be me.

Reply:
I think `confound` is slightly more appropriate here than `confuse` (a replacement would be `mix up`), so I'll keep it here. It might be more formal language, that being said...